aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -323,10 +345,10 @@ export const useChatbot = () => {
     setAbortController(new AbortController());
   };
 
-  const handleSend = async (message: string | number) => {
+  const handleSend = async (query: string | number) => {
```

Comment:
Renamed `message` to `query` because `message` is also used for internal message data in chatbot UI.